tests/testthat/test-helpers.R

```diff
@@ -557,7 +559,7 @@ test_that("h_create_instance works correctly", {
       CohortSizeDLT(dlt_intervals = c(0, 1), cohort_size = c(1, 3))
     )
   )
-  expect_equal(h_create_instance("CohortSizeMax"), test_obj)
+  expect_equal(.DefaultCohortSizeMax(), test_obj)
```

Comment:
now that we have the default constructors coded next to the class definition and user constructor, it would be better for maintenance to also put the small test for each of them next to the tests for the user constructor. That way we can also avoid additional functions defined here in the test script

Reply:
Hi @PuzzledFace would that be possible? not sure if you saw my comment therefore checking :-)

Reply:
> Hi @PuzzledFace would that be possible? not sure if you saw my comment therefore checking :-)

Yes.  It's possible.  Working on it as we speak.  Currently, the tests succeed locally but fail on GitHub.  I am investigating...

Reply:
cool thank you! (so basically moving the tests to the specific `test-Model-class.R` etc. files and in there next to the other tests for the same class)

Reply:
Thanks John. Let me think about it. My main challenge is that I want to minimize any kind of magic going on for tests, examples.

Reply:
Aarrgghh!  The `examples` folder is listed in `.Rbuildignore`, so my testing strategy fails on GitHub because the folder isn't there to provide the expected objects for the automated testing.  So I think I will need to revert to your approach after all...